CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Changelog
 
+## 17.1.0 (12/18/24)
+
+### Access requests support for AWS Identity Center
+
+AWS Identity Center integration will allow users to request short or long term access to permission sets via Access Requests.
```

Comment:
Should change all "will allow" to "now allows" :)

Reply:
@doggydogworld You changed it here but not anywhere below. Those still say "will support ..."